src/jenkins_job_insight/main.py

```
@@ -1354,6 +1354,91 @@ async def analyze_failures(
+
+    result_data = stored["result"]
+    if "request_params" not in result_data:
+        raise HTTPException(
+            status_code=400,
+            detail="Original analysis has no stored request_params; cannot re-analyze",
+        )
+
+    # Reconstruct the original AnalyzeRequest + Settings
+    try:
+        original_body, original_settings = _reconstruct_from_params(result_data)
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)